### DISCUSSION TITLE
About the units of the .STL files

### DISCUSSION THREAD

**Discussion by chadoulis** [STELLA-Landsat/STELLA#20] [Build Notes: How do I make a STELLA?]
When one uploads the _STL_ folders to a 3D Printing Service Website to order the 3D Printed Parts, they are asked about the units of the file i.e., if it is in **mm**, **cm** or **in**. Could you please clarify which are the correct ones?

**Comment by paulmirel** [comment]
We design the parts in Solidworks, in which the native unit is m. I expect that if you use mm that things will turn out the correct sizes, but I haven't actually tried this.

**Comment by chadoulis** [comment]
Just got my 3D-printed parts. The correct dimensions were indeed in mm.
